src/libstd/thread/local.rs

```diff
@@ -106,12 +106,20 @@ pub struct LocalKey<T: 'static> {
 #[stable(feature = "rust1", since = "1.0.0")]
 #[allow_internal_unstable]
```

Comment:
this attribute must be redundant since I was able to simply copy the macro into a playpen, right?